index.js

```diff
@@ -61,5 +67,9 @@ module.exports = function (input, opts) {
 		}
 	});
 
+	extraArgs.forEach(function (extraArgVal) {
+		args.push(String(extraArgVal));
```

Comment:
I think it is clearer to the end user if non-alphanumeric values are appended to the args list as is. In [the original PR](https://github.com/sindresorhus/dargs/pull/20), the type was checked to see if each value was a string, number, or Array.  If the user makes a mistake and includes an object or nested Array in the "_" key, it seems better to let it show in the output than potentially hide incorrect usage.